odoo/addons/base/tests/test_translate.py

```diff
@@ -687,23 +687,31 @@ def _test_create_empty(self, empty_value):
         self.assertEqual(group.with_context(lang='fr_FR').comment, 'French comment 2')
         self.assertEqual(group.with_context(lang='en_US').comment, 'French comment')
 
-    def test_field_selection(self):
-        """ Test translations of field selections. """
+    def test_update_field_translations(self):
         self.env['res.lang']._activate_lang('fr_FR')
-        field = self.env['ir.model']._fields['state']
-        self.assertEqual([key for key, _ in field.selection], ['manual', 'base'])
-
-        ir_field = self.env['ir.model.fields']._get('ir.model', 'state')
-        ir_field = ir_field.with_context(lang='fr_FR')
-        ir_field.selection_ids[0].name = 'Custo'
-        ir_field.selection_ids[1].name = 'Pas touche!'
-
-        fg = self.env['ir.model'].fields_get(['state'])
-        self.assertEqual(fg['state']['selection'], field.selection)
 
-        fg = self.env['ir.model'].with_context(lang='fr_FR').fields_get(['state'])
-        self.assertEqual(fg['state']['selection'],
-                         [('manual', 'Custo'), ('base', 'Pas touche!')])
+        langs = self.env['res.lang'].get_installed()
+        self.assertEqual([('en_US', 'English (US)'), ('fr_FR', 'French / Français')], langs,
+                         "Test did not started with expected languages")
```

Comment:
Most tests in this class rely on this, no need to check it.  Please remove those lines.